docs/config.md

```diff
@@ -173,7 +177,11 @@ profile_models: <comma-delimited-string-list|list|profile_model>
 [ triton_server_flags: <dict> ]
 
 # Allows custom configuration of perf analyzer instances used by model analyzer
-[ perf_analyzer_flags: <dict>]
+[ perf_analyzer_flags: <dict> ]
+
+# Allows custom configuration of the environment variables for tritonserver instances
+# launched by model analyzer
+[ triton_server_environment: <dict> ]
```

Comment:
"triton_server_environment_variables"

Reply:
My logic here was just that wherever we see being able to pass environment variables, the "variables" is always left out. For example docker environment, Popen environment etc. Unless you feel strongly that it should be more specific, I'd rather keep it shorter.

Reply:
That makes sense. Let's keep it this way.